_spinnaker_install_admin_guides/packer.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Understanding Bake Scripts (Packer templates)
+order: 80
+# Substantially different from install_guide/packer
+redirect_from:
+  - /spinnaker_install_admin_guides/packer/
```

Comment:
Should there be a path here for `/spinnaker-install-admin-guides/packer`?

Reply:
Nope, this page is hosted at `/spinnaker-install-admin-guides/packer` and doesn't need a self redirect.